.github/workflows/post-merge-actions-dev.yml

```diff
@@ -52,6 +52,13 @@ jobs:
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@v2
 
+      - name: Login to GDS Dev Dynatrace Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: khw46367.live.dynatrace.com
+          username: khw46367
+          password: ${{ secrets.DYNATRACE_PAAS_TOKEN }}
+
       - name: Build, tag, and push image to Amazon ECR
```

Comment:
"Login to GDS Dev Dynatrace Container Registry" need to be added on post-merge-actions-build.yml

Reply:
Added this block of code to post-merge-actions-build.yaml file

     - name: Login to GDS Dev Dynatrace Container Registry
        uses: docker/login-action@v3
        with:
          registry: khw46367.live.dynatrace.com
          username: khw46367
          password: ${{ secrets.DYNATRACE_PAAS_TOKEN }}